calico_node/tests/st/utils/utils.py

```diff
@@ -223,6 +223,42 @@ def check_bird_status(host, expected):
                   "Output: \n%s" % (ipaddr, peertype, state, output)
             raise AssertionError(msg)
 
+@debug_failures
+def update_bgp_config(host, nodeMesh=None, asNum=None):
```

Comment:
Is it worth having a flag for choosing whether you set BGP config using a `BGPConfigurationList` or just a `BGPConfiguration`?

Reply:
Do we need to add tests where we check the function of multiple `BGPConfiguration`s in a list?

Reply:
I would think tests like that should go into calicoctl... though we do have calicoctl tests in here so maybe?

Reply:
I'd prefer that what Neil and I are working on in the STs right now though is just kept to the scope of getting the current tests working.  This already seems like a big enough task as it is.